src/Kafka/CreateKafkaInstance/CreateKafkaInstance.css

```diff
@@ -9,6 +9,6 @@
 .mk--create-instance-modal__sidebar--content dt {
   font-size: var(--pf-global--FontSize--sm);
 }
-.mas-m-modalTop {
-  top: var(--pf-global--spacer--md);
+.mk--create-instance-modal__alert--message {
+  margin-top: var(--pf-global--spacer--md);
```

Comment:
This is messing up the layout for stories when there is no message being shown. Take the "Default" story for example, the "Name *" label should be visually aligned with the right box.

<img width="887" alt="image" src="https://user-images.githubusercontent.com/966316/159480747-02a57081-a651-4954-9f61-5414e892e3cb.png">

Reply:
I'm not sure I follow what the issue is, or understand how to reproduce it. 

I did notice that the flex layout somehow magically aligns the first line of text in the first flex item with the first line of text in the second flex item. For example, if you remove the padding from the card title so that it shifts up, the Name input also shifts up.

Reply:
Why do you think this alignment issue is related to this css ? This css class used in bottom for inline alert message. 
![image](https://user-images.githubusercontent.com/11775081/159655165-9b78320c-3c58-4e1e-a18e-5af813f4c1b0.png)


I figure out this issue. Details header taking top padding due to `<CardTitle component="h2">` tag. 
`padding-top:var(--pf-c-card--first-child--PaddingTop);`

@jgiardino any suggestion for fixing alignment issue for Name filed with Details box. 
One fix is we can over wright `CardTitle` css class or we can use alternate of  CardTitle.

Reply:
Right, this specific line is unrelated. It's the default alignment of the FlexItem that needs to be tweaked. I'll comment on the right place